src/Results/Results.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import Deputado from '../components/Deputado/Deputado';
+import ProgressBar from '../components/ProgressBar/ProgressBar';
+import './Results.scss';
+
+class Results extends Component {
+  onClickShare(deputado) {
+    console.log(deputado);
+  }
+
+  onClickPlus(deputado) {
+    console.log(deputado);
+  }
+
+  renderBotaoVerMais() {
+    const { temMaisRegistros = false } = this.props;
+
+    if(temMaisRegistros) {
+      return (
+        <div className='barra-ver-mais'>
+          <button className='ver-mais'>Ver mais</button>
+        </div>
+      );
+    }
+  }
+
+  renderCandidatos() {
+    const { candidatos = [] } = this.props;
+
+    return candidatos.map((c, index) => <Deputado key={index} {...c}
+      onClickPlus={this.onClickPlus.bind(this, c)}
+      onClickShare={this.onClickShare.bind(this, c)}/>);
```

Comment:
Essas funções não precisam de bind, o que acha de resolver com arrow functions? Elas receberiam apenas um parâmetro, que seria `() => this.onClickPlus(c)`.